pkg/kv/kvprober/kvprober.go

```diff
@@ -335,8 +357,8 @@ func (p *Prober) Start(ctx context.Context, stopper *stop.Stopper) error {
 					return
 				}
 
-				probeCtx, sp := tracing.EnsureChildSpan(ctx, p.tracer, opName+" - probe")
-				probe(probeCtx, pl)
+				_, sp := tracing.EnsureChildSpan(ctx, p.tracer, opName+" - probe")
```

Comment:
What is the reason for ths change?

Reply:
My bad. I was trying something else and forgot to change this back.